bigcache.go

```diff
@@ -33,6 +33,79 @@ type cacheShard struct {
 	onRemove    func(wrappedEntry []byte)
 }
 
+// EntryInfo holds informations about entry in the cache
+type EntryInfo struct {
```

Comment:
Ok, done, it saves some allocs (20B/op vs 32B/op). Times are the a little smaller.